src/frontend/Komponenter/Personoversikt/Historiskpensjon/Historiskpensjon.tsx

```diff
@@ -28,10 +40,18 @@ const Historiskpensjon: React.FC<{ fagsakPersonId: string }> = ({ fagsakPersonId
                 const harPensjondata = historiskPensjon.harPensjonsdata;
                 return harPensjondata ? (
                     <StyledWarningStripe>
-                        Bruker har saker før desember 2008 som kan sees i{' '}
-                        <a href={historiskPensjon.webAppUrl}>PE PP - Historisk pensjon</a>
+                        <FlexBox>
+                            <BodyShort>Bruker har fått stønad før desember 2008</BodyShort>
+                            <Link href={historiskPensjon.webAppUrl} target={'_blank'}>
+                                <Button type={'button'} as={'p'} size={'small'}>
```

Comment:
Er det ønskelig å ha en button her? Når det skal vises for tidligere vedtaksperioder ska vi kun ha en lenke https://www.figma.com/file/JEmXg4WqG0DFItNGpoX696/Skisser-EF-sak?node-id=8459%3A130283&t=FsXFrW8TxOsZR8cF-0

Reply:
En designvurdering Mirja og jeg tok mens Lars var borte. Skal høre med han på nytt. Mirja "fant ikke" lenken da vi brukertestet på henne... Da er vi isåfall "like langt"...

Reply:
<img width="1307" alt="Skjermbilde 2023-02-01 kl  14 55 02" src="https://user-images.githubusercontent.com/402915/216062487-a7399e23-ff28-4f31-b0d0-aad36e2e2294.png">
 Bedre nå? :)